handlers/handlers.go

```diff
@@ -403,17 +403,29 @@ func (s *ServiceSet) tiles(db *mbtiles.DB) handlerFunc {
 	}
 }
 
+// TileMux returns a pointer to a http.ServeMux that serves the TileJSON
+// information and the tiles for all DBs in the ServiceSet.
+// The function ef is called with any occuring error if it is non-nil, so it
+// can be used for e.g. logging with logging facitilies of the caller.
+func (s *ServiceSet) TileMux(ef func(error)) *http.ServeMux {
+	m := http.NewServeMux()
+	for id, db := range s.tilesets {
+		p := "/services/" + id
+		m.Handle(p, wrapGetWithErrors(ef, s.serviceInfo(id, db)))
```

Comment:
@fawick we should rename `serviceInfo` to `tileJSON` to make its role more explicitly clear.

Reply:
One complication that I'm not yet sure how to solve is that the tileJSON we return contains URLs to endpoints that are provided by `Handler` below.

I wonder - could we make `s.tileJSON` configurable, so that we pass in a boolean indicating if it should also render out a URL to the map endpoint?  (default to false?)

Reply:
Yes, you're right, `tileJSON` is a better name. I am going to rename this.

For the URL are probably referring to value of the `map` key in the output, right? Is the `map` key even in the TileJSON spec? I could not find it when I just searched for it. If it is not part of the spec but an extension by mbtileserver, I'd say it's okay to leave this at it is, because no TileJSON consuming software is expecting this key.

Reply:
It is not part of the spec. The spec is [here](https://github.com/mapbox/tilejson-spec/tree/master/2.1.0).  I'm pretty sure I got the inspiration for adding `map` endpoint to the enhanced tile JSON returned by Mapbox hosted services.